tmt/base.py

```diff
@@ -771,12 +773,20 @@ def _fmf_id(self) -> None:
 
     # TODO: cached_property candidates
     @property
-    def fmf_root(self) -> Path:
-        return Path(self.node.root)
+    def fmf_root(self) -> Optional[Path]:
+        # Check if fmf root exists
+        try:
+            return Path(self.node.root)
+        except TypeError:
+            return None
```

Comment:
I'd recommend checking whether `self.node.root` is `None` or not. It's a simple check, more readable than `try/except`.